corehq/apps/app_manager/models.py

```diff
@@ -1275,15 +1276,15 @@ def case_list_modules(self):
     def is_case_list_form(self):
         return bool(self.case_list_modules)
 
-    def get_save_to_case_updates(self, case_type):
+    def get_save_to_case_updates(self):
```

Comment:
You'll see a pattern in this file of taking methods that used to give all of the properties for a single (specified) case type and changing them to return properties for every case type they stumble upon in the process (as `{<case_type>: set([<property>])}`). This prevents us from having to know all of the case_types in advance and then ask each of these functions to do it for each of the case_types we know about, which cuts down on the number of times we have to execute these loops, but also (and more importantly) makes it easier to reason with.